src/views/studio/CreateEditChannelView/CreateEditChannelView.style.tsx

```diff
@@ -1,9 +1,9 @@
 import styled from '@emotion/styled'
```

Comment:
Get rid of `.tsx` extension, also let's rename it to `CreateEditChannelView.styles.ts`